pubsub/jetstream/jetstream.go

```diff
@@ -153,6 +153,9 @@ func (js *jetstreamPubSub) Subscribe(ctx context.Context, req pubsub.SubscribeRe
 	if v := js.meta.DurableName; v != "" {
 		consumerConfig.Durable = v
 	}
+	if v := js.meta.QueueGroupName; v != "" {
+		consumerConfig.DeliverGroup = v
+	}
```

Comment:
Please create a unit test so we don't have regressions in future. It looks like we don't have _any_ for this code path.

I'd suggest creating a new function `consumerConfigFromMeta`, and create units tests in `jetstream_test.go`.

Reply:
Thanks, will do!

Reply:
@JoshVanL I will not block this PR for unit tests here - but you are right that we should have unit tests for the metadata.

@bruth please try to complete this in the next day or two.